query-engine/query-engine/src/context.rs

```diff
@@ -43,7 +44,7 @@ impl ContextBuilder {
 
 impl PrismaContext {
     /// Initializes a new Prisma context.
-    async fn new(
+    pub async fn new(
```

Comment:
I've searched the diff and found no reference to that function. Is making this public necessary because other code moved? (but that should be visible from the diff).

Reply:
I splitted this PR from a different one, and that might required this function to be public. I will restrict and address if needed in the PR that adds capturing